### PR TITLE
#8151: update sweep test and doc for reglu

### PR DESCRIPTION
**Pull request by bharane-ab**
### Ticket
Link to Github Issue #8151 

### Problem description
Update doc with supported dtype and layout and move sweep test

### What's changed
- Updated document with supported data type and supported layout 
- Updated and moved sweep test

### Checklist
- [x] [Post commit CI passes](https://github.com/tenstorrent/tt-metal/actions/runs/11186626051) - PASSED
- [x] [Run Sweeps](https://github.com/tenstorrent/tt-metal/actions/runs/11186617886)

### CODE REVIEW

File: tests/ttnn/sweep_tests/sweeps/sweeps/reglu.py [hunk 1]
```diff
@@ -17,13 +17,27 @@
...
+    batch_sizes,
+    height,
+    width,
+    input_dtype,
+    input_memory_config,
+    output_memory_config,
+    layout,
+) -> Tuple[bool, Optional[str]]:
+    if layout == ttnn.ROW_MAJOR_LAYOUT:
+        return True, "This combination is not supported"
```
Comment (VirdhatchaniKN) [inline tests/ttnn/sweep_tests/sweeps/sweeps/reglu.py:37]: Can you please change this statement?

**Review by VirdhatchaniKN — APPROVED** [review]
Minor comment

**Review by VirdhatchaniKN — CHANGES_REQUESTED** [review]
Please remove older sweeps and migrate to new infra

**Review by VirdhatchaniKN — APPROVED** [review]
LGTM

**Review by Aswinmcw — APPROVED** [review]
LGTM

**Review by jdesousa-TT — APPROVED** [review]
Please run sweeps pipeline on this branch for reglu before merging.

**Review by eyonland — APPROVED** [review]

**Pull request merged**